src/main/java/com/cinema/services/HallServiceImpl.java

```diff
@@ -28,6 +28,15 @@ public Hall getOne(Long id){
         existsById(id);
         return hallRepository.getOne(id);
     }
+
+    public boolean addHall(Hall hall) {
```

Comment:
should return Id of created Hall rather than boolean

Reply:
you absolutely rite. I fixed it